cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -360,7 +361,8 @@ func (c *cachingMigInfoProvider) fillMigInfoCache() error {
 				templateUrl, err := url.Parse(zoneMig.InstanceTemplate)
 				if err == nil {
 					_, templateName := path.Split(templateUrl.EscapedPath())
-					c.cache.SetMigInstanceTemplateName(zoneMigRef, templateName)
+					regional, _ := regexp.MatchString("(/projects/.*[A-Za-z0-9]+.*/regions/)", templateUrl.String())
```

Comment:
Please put this regexp matching into a helper function, it is used in more than one place.

Reply:
Done